README.md

```diff
@@ -167,7 +167,7 @@ Returns the rank of each stride.
 
 ## is_column_major(A)
 
-Returns a `Val{true}()` if `A` is column major, and a `Val{false}()` otherwise.`
+Returns a `Val{true}()` if `A` is column major, and a `Val{false}()` otherwise.
```

Comment:
Should probably have made these return `True()`/`False()`. Should have caught that before 3.0.